deploy/lvm-operator.yaml

```diff
@@ -1147,6 +1147,14 @@ roleRef:
   name: openebs-lvm-provisioner-role
   apiGroup: rbac.authorization.k8s.io
 
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: openebs-lvm-localpv-csi-controller-critical
+value: 900000000
```

Comment:
what this values means?

Reply:
A PriorityClass object can have any 32-bit integer value smaller than or equal to 1 billion. Larger numbers are reserved for critical system Pods that should not normally be preempted or evicted. The higher the value, the higher the priority.
Above is the max value we can set for custom priorityclass.

Reply:
let us describe this in the comment.

Reply:
added